### DISCUSSION TITLE
Give a visual indicator on scores set in lazer on the website

### DISCUSSION THREAD

**Discussion by cjwrecks** [ppy/osu#26866] [Ideas]
Passes are objectively easier for a lot of maps and a lot of players in lazer. There needs to be some kind of indicator on the score saying "this score was set on lazer" with some kind of icon or something. I think something along the lines of a small icon that you can hover over to view if a score was on lazer or stable would be a useful way of implementing this, considering all this dead space in score cards (you could also just put lazer or stable after the icon instead of as a hover although i think visually that is more readable it also kind of looks tacky so idk)
![image psd](https://github.com/ppy/osu/assets/147895692/f5102d94-8b3b-4a4b-a7f6-f7e4a7b66afa)

**Comment by TextAdventurer12** [comment]
this is what the classic mod is for

**Reply by cdwcgt** [reply]
classic mod score also can be set by lazer
